### ISSUE TITLE
Cannot read property 'viewContainer' of undefined in CdkTree Angular 9 RC3

### ISSUE THREAD

**Issue by jinder** [angular/components#17806]
Since upgrading to Angular 9 RC3, I'm seeing the following exception when creating a CDK Tree:

```
core.js:6068 ERROR TypeError: Cannot read property 'viewContainer' of undefined
    at CdkTree.renderNodeChanges (tree.js:730)
    at SafeSubscriber._next (tree.js:716)
    at SafeSubscriber.__tryOrUnsub (Subscriber.js:183)
    at SafeSubscriber.next (Subscriber.js:122)
    at Subscriber._next (Subscriber.js:72)
    at Subscriber.next (Subscriber.js:49)
    at TakeUntilSubscriber._next (Subscriber.js:72)
    at TakeUntilSubscriber.next (Subscriber.js:49)
    at Observable._subscribe (subscribeToArray.js:3)
    at Observable._trySubscribe (Observable.js:42)
```

This is caused by the _nodeOutlet property on the CdkTree being undefined.

#### Environment

- Angular: 9 RC 3
- CDK/Material: 9 RC 4
- Browser(s): Chrome 78
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by jelbourn** [comment]
@crisbeto could you take a look?

**Comment by crisbeto** [comment]
Nothing stands out that could be causing the issue. Also the tree hasn't been touched in a while so it's unlikely that we've introduced this recently. @jinder are you running with Ivy and is it possible to isolate this into a Stackblitz?

**Comment by jinder** [comment]
I've spent half a day or so trying to isolate the problem to reproduce on a small project, and I can't.  I've cleared my yarn cache, done a fresh install, and then gradually stripped away all the code in our component.  The error occurs when I set the dataSource, regardless of what's in it.  For whatever reason, the ViewChild cdkTreeNodeOutlet isn't being found, even though cdkTreeNodeOutlet is defined in my template.

When I do this on a fresh project, there's no issue - but I'm not sure what set of code/dependencies would cause the Ivy ViewChild behaviour to change for our specific project.  I think I'll wait for the next RC and see how that behaves.
